### PR TITLE
updated pkgr to latest MPN snapshot (for mrgvalprep and mrgvalidate)

### PR DESCRIPTION
**Pull request by barrettk**
Update pkgr to have latest MPN snapshot. mrgvalprep had a bug in it that has since been addressed

### CODE REVIEW

File: inst/validation/bbr-requirements.yaml [hunk 1]
```diff
@@ -619,7 +619,7 @@ NWMD-R010:
   tests:
   - BBR-NWMD-010
 NWMD-R011:
-    description: new_model() ignores files' extensions
+  description: new_model() ignores files' extensions
```
Comment (kyleam) [inline inst/validation/bbr-requirements.yaml:622]: Could you drop this change?  Let's keep this PR restricted to the simple MPN bump.

Comment (barrettk) [reply inst/validation/bbr-requirements.yaml:622]: Is there an easy way to remove that commit?

Comment (kyleam) [reply inst/validation/bbr-requirements.yaml:622]: pinging you for call

**Review by kyleam — APPROVED** [review]
Thanks, good to go once drone completes.

**Pull request merged**